build/Versions.props

```diff
@@ -36,7 +36,7 @@
     <NuGetProjectModelVersion>$(NuGetBuildTasksPackVersion)</NuGetProjectModelVersion>
     <PlatformAbstractionsVersion>2.0.0</PlatformAbstractionsVersion>
     <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
-    <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
+    <SystemReflectionMetadataVersion>1.6.0-preview2-26406-04</SystemReflectionMetadataVersion>
```

Comment:
There's risk of shipping a random prerelease here unless we are getting dependency uptake from prodcon. Are we? Using RTM seems better if not.

Reply:
@ericstj @nguerrera Agreed we should figure out if we can depend on an RTM version.  The problem with 1.5.0 is that it's not multi-targeted to .NET Framework, so it will need the .NET Standard facades, which means more risk of having loading issues with it in the MSBuild task context.

Reply:
Looks like we'll pick up the netstandard1.1 from net46 ref, which should only require in-box facades.

Reply:
Correct.  You shouldn't see any facades deployed for this.